master/internal/provisioner/aws_config.go

```diff
@@ -31,6 +31,19 @@ type AWSClusterConfig struct {
 	LogStream string `json:"log_stream"`
 }
 
+var defaultAWSImageID = map[string]string{
+	"us-west-2": "ami-00f470c0ed034f8e7",
```

Comment:
dashes or hyphens?

Reply:
I think they are hyphens, those you could type with the bare keyboard. I copy-pasted those image ids. Why do you ask that?

Reply:
Ahh I just realized I mixed the use of them. NVM. I'll fix them